scripts/generate_package_docs.py

```diff
@@ -0,0 +1,98 @@
+from sys import argv
+import json
+import re
+import utils
+
+config_location = "config/conf.json"
+
+def main():
+  params = ParamProcessor(argv)
+  write_docs_to_bucket(params)
+
+def write_docs_to_bucket(params):
```

Comment:
I like being able to see what parameters are used by a method by looking at the method's signature, which is difficult to do when all `params` are passed in. It'd also allow for including types in method signatures...which is a good segue to plug [type hints](https://docs.python.org/3/library/typing.html) and [mypy](http://mypy-lang.org/). Again, because these may be stand-alone scripts, they may not be a big deal but...

Reply:
i implemented all the typing hints only to learn that this became a thing in python 3.5 so I can't since I had to revert to 2.7 for jenkins :(

Reply:
Bummer :( Could still spell out the needed subset of arguments but not a blocker.